src/main/java/me/ironexception/universalfriends/association/Association.java

```diff
@@ -38,8 +41,14 @@ public double getValue() {
     /**
      * Fetch a {@link Association} by its value.
      * Might return null if there is no corresponding
-     * @param value
-     * @return
+     *
+     * <p>
+     * This is like the friendliness level.
```

Comment:
It's not just *like* the friendliness level, it really **is** the friendliness *value*.